.github/workflows/test.yml

```diff
@@ -112,14 +112,14 @@ jobs:
       matrix:
         go: ['1.18.10', '1.19.12', '1.20.7', '1.21.0']
     name: Test with Go ${{ matrix.go }} on FreeBSD
-    runs-on: macos-12
+    runs-on: ubuntu-22.04
     defaults:
       run:
         shell: bash
     steps:
       - uses: actions/checkout@v3
       - name: Run in freebsd
-        uses: vmactions/freebsd-vm@v0
+        uses: vmactions/freebsd-vm@v1
         with:
           usesh: true
           prepare: |
```

Comment:
The comment in the run section should be updated since the FreeBSD vm runs in qemu not macOS in virtual box. Otherwise, LGTM